src/platform/packages/shared/kbn-esql-ast/src/parser/constants.ts

```diff
@@ -13,3 +13,59 @@ export const DEFAULT_CHANNEL: number = +(Token as any).DEFAULT_CHANNEL;
 export const HIDDEN_CHANNEL: number = +(Token as any).HIDDEN_CHANNEL;
 
 export const SOURCE_COMMANDS = new Set<string>(['FROM', 'ROW', 'SHOW', 'TS', 'EXPLAIN']);
+
+export const PROCESSING_COMMANDS = new Set<string>([
```

Comment:
Maybe I am missing something but I was wondering if it wouldn't be better to try to have a single source of truth, perhaps using information in our esqlCommandRegistry 

example:
```javascript 
export const SOURCE_COMMANDS = .....

export const PROCESSING_COMMANDS = new Set<string>(
  esqlCommandRegistry.getAllCommandNames()
    .map(name => name.toUpperCase())
    .filter(name => !SOURCE_COMMANDS.has(name)) // or just map all commands and remove this line
);
export const COMMANDS = ....;```

Reply:
yes this is a good idea. I was also wondering if we can get this info from ANTLR

Reply:
Agreed